tip-distributor/src/lib.rs

```diff
@@ -494,14 +499,17 @@ pub async fn sign_and_send_transactions_with_retries(
                 });
         }
 
-        let futs = signatures_to_transactions
-            .iter()
-            .map(|(sig, tx)| async move {
+        let futs = signatures_to_transactions.iter().map(|(sig, tx)| {
```

Comment:
wb single threading sends here (sends are over localhost so super fast) then single threading confirmations (same wrt speed)

Reply:
alternatively, could limit number of tokio worker threads

Reply:
and not use semaphore

Reply:
could do:
```
let mut rt = runtime::Builder::new()
    .core_threads(4)
    .build()
    .unwrap();

```